microsoft/testsuites/core/storage.py

```diff
@@ -374,6 +376,63 @@ def verify_azure_file_share_nfs(self, log: Logger, node: Node) -> None:
         finally:
             nfs.delete_share()
             node.tools[NFSClient].stop(mount_dir)
+    
+    _file_path = "~/SAMPLE.txt"
+    _original_text = "Hello, world!"
+    _mount_point = "/mnt/data0"
+
+    @TestCaseMetadata(
+        description="""
+        This test case will verify mount azure nfs on guest successfully.
+        1. Write “Hello World!” string to a file. 
+        2. Move the file from home folder to data disk. 
+        3. Assert the file content in data disk. 
+        4. Assert the file on original place doesn't exist. 
+        """,
+        timeout=TIME_OUT,
+        priority=4,
+        requirement=simple_requirement(
+            disk=AzureDiskOptionSettings(
+                data_disk_count=search_space.IntRange(min=1),
+                data_disk_size=search_space.IntRange(min=20),
+            ),supported_platform_type=[AZURE],
```

Comment:
We do not need to limit Platform to Azure, since no Azure dependent tools were used. This test seems like something that can run on any other cloud platforms as well.

Reply:
Removed the requirement